pkg/stream/constants.go

```diff
@@ -135,6 +141,12 @@ func lookErrorCode(errorCode uint16) error {
 		return FrameTooLarge
 	case responseCodeAccessRefused:
 		return CodeAccessRefused
+	case responseCodeStreamNotAvailable:
```

Comment:
Nice! I think I did see some "unknown response code" errors or something like that, will be helpful to have these.